app/assets/javascripts/discourse/app/controllers/full-page-search.js

```diff
@@ -383,6 +384,9 @@ export default Controller.extend({
               this.set("model", model);
             }
           })
+          .catch((e) => {
+            this.set("error", e.jqXHR.responseJSON?.message);
```

Comment:
Should a successful response reset the error property? Another way to ask this is: what happens if after an error request, users makes a new one that returns results, does the error message get cleared from the UI? (It should.)